uwuno/src/main/java/com/learning/uwuno/services/gameService.java

```diff
@@ -23,20 +23,15 @@
     // Class Functions
     // POSTS
     public room addRoom(String roomName, boolean useBlankCards) {
-        if (roomName.isEmpty())
+        if (roomName.isBlank())
             throw new badRequest();
         room newRoom = new room(roomName, useBlankCards);
         roomList.add(newRoom);
         return newRoom;
     }
 
-    public String addRoom(room newRoom) {
-        roomList.add(newRoom);
-        return newRoom.getUid();
-    }
-
     public player addPlayer(String name, String uid) {
-        if (name.isEmpty())
+        if (name.isBlank())
```

Comment:
This change is also in my branch already.
I think you forgot to merge my current branch into yours before starting the fixes.

Reply:
I just merged mine into master, will have some conflicts you'll need to fix before testing everything again :(

Reply:
Yup. Done.